dataactcore/models/lookups.py

```diff
@@ -0,0 +1,90 @@
+# This file defines a series of constants that represent the values used in the
+# broker's "helper" tables. Rather than define the values in the db setup scripts
+# and then make db calls to lookup the surrogate keys, we'll define everything
+# here, in a file that can be used by the db setup scripts *and* the application
+# code.
+# todo: replace getIdFromDict and getNameFromDict baseInterface functions with these constants
+
+from collections import namedtuple
+
+LookupType = namedtuple('LookupType', ['id', 'name', 'desc'])
+LookupFileType = namedtuple('LookupFileType', ['id', 'name', 'desc', 'letter', 'order'])
```

Comment:
This bit of confusion goes back to the fact that we have two file type tables (an artifact from the days of multiple dbs). I'm using `LookupFileType` as a way to load both (`file_type` and `file_type_validation`) rather than pull the thread or merging those (which we should do at some point).